content/de/usage/setup.md

```diff
@@ -42,7 +43,7 @@ Unsere Bibliotheken funktionieren mit physischen Handys und Emulatoren unter And
 | --- | --- | --- |
 | Android | `device` (Moto G7 Power) | 13 (API-Level 33) |
 | Android | `emulator` | 11 (API level 30), 13 (API-Level 33) |
-| iOS | `device` (iPhone X, iPhone 6S) | 15.6.1, 16.0 |
+| iOS | `device` (iPhone X, iPhone 6S) | 15.7.5, 16.3.1 |
```

Comment:
I'm still testing with 15.6.1 and 16.0, so I think we should still mention those.

```suggestion
| iOS | `device` (iPhone X, iPhone 6S) | 15.6.1, 15.7.5, 16.0, 16.3.1 |
```